tox.ini

```diff
@@ -80,6 +80,7 @@ commands =
 
 
 [testenv:codechecks]
+basepython = python2.7
```

Comment:
Why do we need this?

Reply:
Did you see the commit message, d0d11a5444d52b4ead4165ad2b418943da196b91?  Anything I can clarify from that?

Reply:
Ah, I did not, thanks. Makes sense :+1: